mrblib/xremap/key_expression.rb

```diff
@@ -38,7 +38,11 @@ def modifier_mask(modifiers)
       end
 
       def to_keysym(keyexp)
-        X11.const_get("XK_#{keyexp}")
+        begin
+          X11.const_get("XK_#{keyexp}")
+        rescue NameError # Allow non "XK_" expressions
```

Comment:
Could you switch the behavior by `if keyexp.start_with?('XF86XK')` rather than branching by exception?